src/main/java/edu/kis/powp/jobs2d/command/manager/ISourceToDraw.java

```diff
@@ -0,0 +1,10 @@
+package edu.kis.powp.jobs2d.command.manager;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+
+import java.util.List;
+
+public interface ISourceToDraw {
```

Comment:
Wrong name. It is not about drawing but generating commands (drawing is only a simulation of a device work)